circle/docker-release-image.sh

```diff
@@ -25,13 +25,30 @@ if [[ -n $RELEASE_SERIES_LIST && -z $LATEST_STABLE ]]; then
 fi
 IFS=',' read -ra RELEASE_SERIES_ARRAY <<< "$RELEASE_SERIES_LIST"
 
+IS_DEFAULT_PLATFORM=1
+OS_PLATFORM=""
+if ! is_default_platform "$IMAGE_TAG" ; then
+  IS_DEFAULT_PLATFORM=0
+  OS_PLATFORM="$(get_os_platform "${IMAGE_TAG}")"
+  if [[ "${OS_PLATFORM}" = "rhel-"* ]]; then
+    info "Skipped publishing ${OS_PLATFORM} image: RHEL releases are disabled."
+    exit 0
+  fi
+fi
+
 MATCHING_RS_FOUND=0
 for RS in "${RELEASE_SERIES_ARRAY[@]}"; do
   if [[ "$IMAGE_TAG" == "$RS"* ]]; then
-    CACHE_TAG=$RS
-    RELEASE_SERIES=$RS
     let MATCHING_RS_FOUND+=1
-    TAGS_TO_UPDATE+=($RELEASE_SERIES)
+    RELEASE_SERIES="${RS}"
+    CACHE_TAG="${RELEASE_SERIES}"
+
+    if [ "${IS_DEFAULT_PLATFORM}" == 0 ] ; then
+        RELEASE_SERIES+="/${OS_PLATFORM}"
```

Comment:
why there is a slash here? this will be `1.15/rhel-7` for nginx for example no? Is it the path to the Dockerfile?

Reply:
Yes, exactly. `RELEASE_SERIES` is being used in `docker_build` as the path where to find the `Dockerfile`.